rave/src/main/java/com/uber/rave/Rave.java

```diff
@@ -69,6 +70,34 @@ public static synchronized Rave getInstance() {
     }
 
     /**
+     * Validate elements in a collection.
+     * @param collection the collection to validate
+     * @param <T> the type of the collection
+     * @throws RaveException if validation fails.
+     */
+    public <T> void validate(Collection<T> collection) throws RaveException {
```

Comment:
Do we need new APIs for this? I initially thought we'd just add this to the existing `validate()`.